coalib/bearlib/languages/documentation/default.coalang

```diff
@@ -2,17 +2,23 @@
 doc-marker = """, , """
 param_start = :param\ # here's a space
 param_end = :
+exception_start = :raises\ # here's a space
+exception_end = :
 return_sep = :return:
 
 [PYTHON3]
 doc-marker = """, , """
 param_start = :param\ # here's a space
 param_end = :
+exception_start = :raises\ # here's a space
+exception_end = :
 return_sep = :return:
 
 [JAVA]
 doc-marker1 = /**, \ *, \ */
 doc-marker2 = /**, , \ */
 param_start = @param\ \ # here's a space
 param_end = \ # here's a space
+exception_start = @raises\ \ # here's a space
```

Comment:
Since you are modifying this, you should also modify test data `default.java` and test case.